### PR TITLE
Alias formats without generation number to Gen7 formats

### PR DESCRIPTION
**Pull request by xfix**

### CODE REVIEW

**Comment by Zarel** [comment]
It's too early for this.

**Comment by xfix** [comment]
Fair enough. I assumed that considering OU no longer refers to [Gen 6] for about 10 days, it would be fine to replace this. Merge when ready I suppose?

**Comment by Zarel** [comment]
It'll be a merge conflict by the time it's ready; I'm refactoring `getFormat` right now.

**Comment by TheImmortal** [comment]
bump?

**Comment by Zarel** [comment]
Should probably be done in `getFormat` or something, anyway.

**Comment by Zarel** [comment]
But yes, I think it's fine now.

File: sim/dex.js [hunk 1]
```diff
@@ -495,13 +495,18 @@ class ModdedDex {
 		}
 		name = (name || '').trim();
 		let id = toId(name);
+		const Formats = this.data.Formats;
 		if (this.data.Aliases[id]) {
 			name = this.data.Aliases[id];
 			id = toId(name);
 		}
+		if (!Formats.hasOwnProperty(id) && Formats.hasOwnProperty(`gen7${id}`)) {
+			name = `[Gen 7] ${name}`;
```
Comment (Zarel) [inline sim/dex.js]: This seems unnecessary; it'll be overridden by the `name` entry in Formats.

**Pull request closed without merging**